src/clj/ataru/applications/application_store.clj

```diff
@@ -29,8 +31,10 @@
                                 :key            (str (java.util.UUID/randomUUID))
                                 :lang           (:lang application)
                                 :preferred_name (find-value-from-answers "preferred-name" answers)
-                                :last_name      (find-value-from-answers "last-name" answers)
-                                :content        {:answers answers}}
+                                :last_name (find-value-from-answers "last-name" answers)
```

Comment:
:key :lang and :preferred-name are indendent so that values are aligned, these new ones are not (C-c space in emacs ;) )